src/components/WebexMeeting/WebexMeeting.scss

```diff
@@ -11,4 +11,9 @@
   min-width: $WEBEX_MEETING_MIN_WIDTH;
 
   background: var(--wxc-meeting-background);
+
+  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-in-meeting,
```

Comment:
the meeting component should send an optional class to these 2 sub-components with flex: 1, eg:
```jsx
<WebexInMeeting className="inner-meeting" ... />
```

and in meeting.scss:
```scss
  .inner-meeting {
    flex: 1;
  }
```

this way, the meeting component does not need to know what is the base class name of in meeting and interstitial meeting.